src/App/App.tsx

```diff
@@ -89,24 +100,24 @@ class App extends Component<Props, State> {
               date={ this.state.selectedUSAState.date }
             />
           </section>
-          <section>
+          <section className='stats-container'>
             <Stat 
-              icon={ GiCoffin }
+              icon={ RiVirusFill }
               number={ this.state.selectedUSAState.positive }
               title={ 'Cases' }
-              details={ 'Some details that we want to display' }
+              details={ `This represents ${((this.state.selectedUSAState.positive / this.state.allUSAData.positive) * 100).toFixed(1)}% of all cases.` }
```

Comment:
We should continue looking into how to format these numbers just as you were able to do in the USA overview.